pkg/route/bing_planner.go

```diff
@@ -81,6 +81,10 @@ func (p *bingPlanner) Zip5TransitDistance(source string, destination string) (in
 	return zip5TransitDistanceHelper(p, source, destination)
 }
 
+func (p *bingPlanner) Zip3TransitDistance(source string, destination string) (int, error) {
```

Comment:
I think this is a nit but I thought our pre-commit enforced that all public functions are documented. 
```suggestion
// Zip3TransitDistance does something ...
func (p *bingPlanner) Zip3TransitDistance(source string, destination string) (int, error) {
```

Reply:
I'm happy to add this but I can confirm I did not get a pre-commit warning.